distributions/aws/examples/storage/efs/README.md

```diff
@@ -3,9 +3,9 @@
 This guide describes how to use Amazon EFS as Persistent storage on top of an existing Kubeflow deployment.  
```

Comment:
how is this different compared to guide in  distributions/aws/examples/storage/fsx-for-lustre/README.md  ?

Reply:
This one is the original EFS README from the previous PR. 
Or are you asking what is the difference in the content of the EFS vs the FSx READMEs and if we can combine the two ?

Reply:
you should have not mixed this two into one pr.

Reply:
It was kind of unavoidable in this case since I wanted to reuse files from the training sample of EFS in the FSx PR but I will try and keep it separate to whatever extent possible in the future.